index.js

```diff
@@ -924,6 +921,8 @@ Object.defineProperty(module.exports, 'minimumBytes', {value: 4100});
 
 module.exports.stream = readableStream => new Promise((resolve, reject) => {
 	try {
```

Comment:
You don't need to try/catch here. Any errors thrown in the `Promise` constructor makes the promise reject.